hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -125,6 +126,11 @@
       .withAlternatives("hoodie.table.rt.file.format")
       .withDocumentation("Log format used for the delta logs.");
 
+  public static final ConfigProperty<String> LOG_BLOCK_TYPE = ConfigProperty
```

Comment:
yes. it will be a write config. We can to facilitate newer file groups written out as avro from say Kafka Connect, while a delete job adds bulk deletes/updates into log blocks in parquet.